source/common/tcp_proxy/tcp_proxy.h

```diff
@@ -114,17 +116,29 @@ class TunnelingConfigHelperImpl : public TunnelingConfigHelper {
 public:
   TunnelingConfigHelperImpl(
       const envoy::extensions::filters::network::tcp_proxy::v3::TcpProxy_TunnelingConfig&
-          config_message)
-      : hostname_(config_message.hostname()), use_post_(config_message.use_post()),
-        header_parser_(Envoy::Router::HeaderParser::configure(config_message.headers_to_add())) {}
-  const std::string& hostname() const override { return hostname_; }
+          config_message,
```

Comment:
nit: It's better to move this implementation code to `.cc` file to prevent header pollution. also `this->` is unusual, but not a big deal.

Reply:
Ok, I moved constructor and `host` method to .cc file.